mangaki/mangaki/views.py

```diff
@@ -696,13 +694,6 @@ def add_pairing(request, artist_id, work_id):
     return HttpResponse()
 
 
-@receiver(user_signed_up)
-@receiver(social_account_added)
```

Comment:
I don't think so. In any case, the `Profile` has nothing to do with the existence of a social account.